de/site.json

```diff
@@ -4,8 +4,8 @@
   "navigation": {
     "TC39": {
       "title": "TC39",
-      "agenda": "Meeting Agendas",
-      "notes": "Meeting Notes",
+      "agenda": "Meeting-Agenden",
+      "notes": "Meeting-Notizen",
```

Comment:
I would translate the agendas as "Tagesordnungen" because a "meeting agenda" is a "Tagesordnung".

I can understand the notion to keep the reference to the meetings and be consistent with "Meeting-Notizen" - and "Meeting" is perfectly fine in German.

I think "Meeting-Notizen" could also be "Mitschriften", **if** we go away from "Meeting-*".

Reply:
Good suggestions. I've struggled quite a bit to come up with a good translation for agenda, as that's simply how we call it at my workplace aswell.

Reply:
We do too. :)